packages/store/src/services/store/errors/errors.test.ts

```diff
@@ -62,11 +62,11 @@ describe('OperationError', () => {
 describe('Error message generation', () => {
   test('should generate correct messages for all error codes', () => {
     // Validation errors
-    expect(new ValidationError(ErrorCodes.FILE_TOO_LARGE, {sizeGB: 6}).message).toBe(
-      'File is too large (6GB). Maximum size is 5GB.',
-    )
-    expect(new ValidationError(ErrorCodes.EMPTY_FILE, {filePath: '/test.db'}).message).toBe('File is empty: /test.db')
-    expect(new ValidationError(ErrorCodes.NOT_A_FILE, {filePath: '/test'}).message).toBe('Path is not a file: /test')
+    expect(
+      new ValidationError(ErrorCodes.FILE_TOO_LARGE, {filePath: '/test.db', fileSize: '5MB', maxSize: '4MB'}).message,
+    ).toBe('File "/test.db" (5MB) exceeds maximum size of 4MB.')
```

Comment:
```suggestion
      new ValidationError(ErrorCodes.FILE_TOO_LARGE, {filePath: '/test.db', fileSize: '5 MB', maxSize: '4 MB'}).message,
    ).toBe('File "/test.db" (5 MB) exceeds maximum size of 4 MB.')
```
(for consistency with my suggestion elsewhere - I realize this test will pass without the change because we do the formatting before passing to this error, although I wonder if we should actually just pass in the number of bytes to this)